internal/cmd/prune_agent.go

```diff
@@ -0,0 +1,47 @@
+/*
+ *  *******************************************************************************
+ *  * Copyright (c) 2019 Edgeworx, Inc.
+ *  *
+ *  * This program and the accompanying materials are made available under the
+ *  * terms of the Eclipse Public License v. 2.0 which is available at
+ *  * http://www.eclipse.org/legal/epl-2.0
+ *  *
+ *  * SPDX-License-Identifier: EPL-2.0
+ *  *******************************************************************************
+ *
+ */
+
+package cmd
+
+import (
+	prune "github.com/eclipse-iofog/iofogctl/internal/prune/agent"
+	"github.com/eclipse-iofog/iofogctl/pkg/util"
+	"github.com/spf13/cobra"
+)
+
+func newPruneAgentCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "agent NAME",
+		Short: "Prune docker images on Agent",
```

Comment:
All the images ? Only dangling ones ? Not clear

Reply:
updated